.github/workflows/scripts/revert-latest.sh

```
@@ -0,0 +1,77 @@
+    echo "🔄 Attempt $attempt/$max_retries: Syncing $source_path to $dest_path"
+
+    if aws s3 sync "$source_path" "$dest_path" \
+       --endpoint-url "$R2_ENDPOINT" \
+       --profile "${R2_AWS_PROFILE:-R2}" \
+       --no-progress \
+       --delete; then
+      echo "✅ Sync successful from $source_path to $dest_path"
+      return 0
+    else
+      echo "⚠️ Attempt $attempt failed"
+      if [ $attempt -lt $max_retries ]; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)